lib/ui/alert_detail.dart

```diff
@@ -27,14 +25,14 @@ class _AlertDetailPageState extends State<AlertDetailPage> {
   final _formKey = GlobalKey<FormState>();
   String title = '';
   String description = '';
-  DateTime date = DateTime.now();
   bool isImportant = false;
   bool needToRepeat = false;
   int daysToRepeat = 0;
   int weekToRepeat = 0;
   int minutesToRepeat = 0;
   int hoursToRepeat = 0;
   DateTime nextNotifyDate = DateTime.now();
+  TimeOfDay nextNotifyTime = TimeOfDay.now();
```

Comment:
Does `DateTime` already include the hour and second? Can we just have one variable of DateTime?